app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
```

Comment:
Do you think this would work as a good getting started, since none of the code in python would be changed? Maybe we can have a template or something so that a portion on the code or logic is in python? 

Otherwise, just having NGINX would be enough, since all you're changing is html.

Reply:
We can change the `name` variable, or add more logic here if necessary. For example, to get the ip of the visitor.
The idea is not to edit an html file. I have had two times this question:
*does okteto only work for ui developers*?

Reply:
I like that. Maybe like the voting app and change the name, title and / or ip?

Reply:
or we could do something like a small game, or a todo app instead of just static html

Reply:
Or this? https://docs.djangoproject.com/en/2.1/intro/tutorial01/

Reply:
What about doing the voting app without persistence?
You click on the vote and it counts the total votes.
Then, you can edit to change Cats by Pets, for example.

Reply:
That's cool too. Whatever is the less amount of work 👍